downstream/modules/platform/con-building-an-execution-environment-in-a-disconnected-environment.adoc

```diff
@@ -16,33 +16,11 @@ link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_pl
 
 * registry.redhat.io for access to the base container images.
 
-* Red Hat does not have an IP list for the container registry domains or cdn-ubi.redhat.com as for cdn.redhat.com. 
-To firewall off the registry and/or UBI repositories, use a proxy (e.g. squid) and allow-list the following domains:
-
-//* `registry.access.redhat.com`
-* `registry.redhat.io`
-//* `registry.connect.redhat.com`
-* `sso.redhat.com`
-* `cdn-ubi.redhat.com`
-* `cdn.quay.io`
-* `cdn01.quay.io`
-* `cdn02.quay.io`
-* `cdn03.quay.io`
-
-For additional information, see link:https://access.redhat.com/articles/1525183[Public CIDR Lists for Red Hat].
+Building an execution environment image in a disconnected environment requires mirroring content from these locations. See link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/red_hat_ansible_automation_platform_installation_guide/index#importing-collections-into-private-automation-hub_disconnected-installation[Importing Collections into private automation hub] for information on importing collections from Ansible Galaxy or automation hub into a private automation hub.
 
 Mirrored PyPI content once transferred into the disconnected network can be made available using a web server or an artifact repository like Nexus.  The RHEL and UBI repository content can be exported from an internet-facing Red Hat Satellite server, copied into the disconnected environment, then imported into a disconnected Satellite so it is available for building custom {ExecEnvShort}s.  See link:https://access.redhat.com/documentation/en-us/red_hat_satellite/6.14/html-single/installing_satellite_server_in_a_disconnected_network_environment/index#iss_export_sync_in_an_air_gapped_scenario[ISS Export Sync in an Air-Gapped Scenario] for details.
```

Comment:
I think I see extra spaces at the end of each sentence in this paragraph--if not please disregard.